tensorflow/lite/tools/serialization/writer_lib.cc

```diff
@@ -69,6 +70,11 @@ TfLiteStatus WriteImpl(const std::string& filename, void* data, size_t size) {
   FILE* fp = fopen(filename.c_str(), "wb");
   if (!fp) return kTfLiteError;
 
+  if (!FLATBUFFERS_LITTLEENDIAN){
```

Comment:
Who does define this?

Reply:
`FLATBUFFERS_LITTLEENDIAN` is defined in the `flatbuffers` header file here: https://github.com/google/flatbuffers/blob/6f895f54c25aa19f5d84ac6cf7fa8bc955a14e1d/include/flatbuffers/base.h#L119-L139

Reply:
Isn't it better to use "#ifdef FLATBUFFERS_LITTLEENDIAN" ?

Reply:
Yes, I'll adjust the code accordingly. Thanks!

Reply:
Done.